src/apps/openfluid-devstudio/MainWindow.cpp

```diff
@@ -456,87 +526,114 @@ void MainWindow::createMenus()
 // =====================================================================
 
 
-void MainWindow::setWorkspaceDefaults()
+void MainWindow::updateBuildJobsStatus()
 {
-  auto* WMgr = openfluid::base::WorkspaceManager::instance();
+  QString JobStatusString;
+  if (m_WareBuildOptions.m_IsParallelJobs)
+  {
+    JobStatusString = QString::fromStdString(std::to_string(m_WareBuildOptions.m_JobsNumber))+" "+tr("jobs");
+    mp_WidgetsCollection->setBuildJobs(m_WareBuildOptions.m_JobsNumber);
+  }
+  else
+  {
+    JobStatusString = tr("No parallel jobs");
```

Comment:
Text to use : " parallel build disabled"

Click to enable/disable parallel build 
parallel build can be enabled only if configured or possible jobs >= 2
when enabled, jobs number can be adjusted using the mouse wheel in the range [2,+inf[